packages/dds/tree/src/tree/anchorSet.ts

```diff
@@ -212,6 +212,8 @@ export class AnchorSet {
                 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
                 srcChildren[index].parentIndex < srcStart!.parentIndex + count
             ) {
+                // Sever the child -/> parent connection
```

Comment:
Is this supposed to be:
```suggestion
                // Sever the child -> parent connection
```